vhdbuilder/packer/packer_source.sh

```diff
@@ -112,6 +112,40 @@ copyPackerFiles() {
   PVT_HOST_SVC_DEST=/etc/systemd/system/reconcile-private-hosts.service
   cpAndMode $CSE_REDACT_SRC $CSE_REDACT_DEST 600
 
+  if grep -q "kata" <<< "$FEATURE_FLAGS"; then
+    IGVM_DEBUG_MEASUREMENT_SRC=/home/packer/igvm-debug-measurement
+    IGVM_DEBUG_MEASUREMENT_DEST=/opt/confidential-containers/share/kata-containers/igvm-debug-measurement
+    cpAndMode $IGVM_DEBUG_MEASUREMENT_SRC $IGVM_DEBUG_MEASUREMENT_DEST 0755
+
+    IGVM_DEBUG_BIN_SRC=/home/packer/igvm-debug.bin
+    IGVM_DEBUG_BIN_DEST=/opt/confidential-containers/share/kata-containers/igvm-debug.bin
+    cpAndMode $IGVM_DEBUG_BIN_SRC $IGVM_DEBUG_BIN_DEST 0755
+
+    IGVM_MEASUREMENT_SRC=/home/packer/igvm-measurement
+    IGVM_MEASUREMENT_DEST=/opt/confidential-containers/share/kata-containers/igvm-measurement
+    cpAndMode $IGVM_MEASUREMENT_SRC $IGVM_MEASUREMENT_DEST 0755
+
+    IGVM_MEASUREMENT_COSE_SRC=/home/packer/igvm-measurement.cose
+    IGVM_MEASUREMENT_COSE_DEST=/opt/confidential-containers/share/kata-containers/igvm-measurement.cose
+    cpAndMode $IGVM_MEASUREMENT_COSE_SRC $IGVM_MEASUREMENT_COSE_DEST 0755
```

Comment:
On top of this, can we create a file `reference-info-base64` that contains the base64-encoded content of `igvm-measurement.cose`?

Reply:
As discussed offline, instead let's just import the `reference-info-base64` file from the upstream pipeline.